docker/docker-entrypoint.sh

```diff
@@ -9,12 +9,19 @@ DB_WAIT_TIMEOUT=${DB_WAIT_TIMEOUT-3}
 MAX_DB_WAIT_TIME=${MAX_DB_WAIT_TIME-30}
 CUR_DB_WAIT_TIME=0
 
+# set NAUTOBOT_DOCKER_ALLOW_UI_BUILD_FAILURE to NAUTOBOT_DOCKER_ALLOW_UI_BUILD_FAILURE, defaulting to "":
+# TODO: Move to just a true/false flag
+UI_BUILD_FAILURE_FLAG=""
+if [[ "$NAUTOBOT_DOCKER_ALLOW_UI_BUILD_FAILURE" == "true" ]]; then
+  UI_BUILD_FAILURE_FLAG="--allow-ui-build-failure"
+fi
+
```

Comment:
In development mode (with NodeJS container) the backend fails to start if the UI fails to build. If you are in-flight with making front-end changes this becomes a bit of a chicken-egg problem because you want the NodeJS container up so you can validate the changes but the backend is down and won't answer your API calls.

Reply:
I'm not a big fan of adding a developer-workflow-specific flag to the docker-entrypoint that's used across the board. Maybe instead a `skip-rebuild-ui` flag or something along those lines might be more generally useful?

Reply:
I think @gsnider2195 pointed out to me "why don't you just skip `post_upgrade`" which might be sufficient anyway.